public/pages/AnomalyCharts/containers/AnomalyDetailsChart.tsx

```diff
@@ -137,7 +136,9 @@ export const AnomalyDetailsChart = React.memo(
     const [zoomRange, setZoomRange] = useState<DateRange>({
       ...props.dateRange,
     });
-    const [zoomedAnomalies, setZoomedAnomalies] = useState<any[]>([]);
+    const [zoomedAnomalies, setZoomedAnomalies] = useState<AnomalyData[][]>(
+      props.anomalies
+    );
 
     const [aggregatedAnomalies, setAggregatedAnomalies] = useState<any[]>([]);
```

Comment:
nit: seems like this can be changed from `useState<any[]>` to `useState<anomalyData[]>`. Seems to only be set as that type, not sure if changing will cause further issues down the line when using the variable.

Reply:
I had changed it from `any[]` to `AnomalyData[][]`, it needs to be a 2D arr now. Not sure what you mean, any other concerns?

Reply:
My mistake I mean to highlight line 143 `const [aggregatedAnomalies, setAggregatedAnomalies] = useState<any[]>([]);` but also it's not something you seem to change this PR so probably unnecessary for now.

Reply:
Good catch actually, added type safety, verified results filtering/aggregation working as expected.